frontend/tests/frontend/helloworld.js

```diff
@@ -0,0 +1,9 @@
+import { helloWorld } from "../../src/js/helloworld";
+
+/* jshint undef: false */
```

Comment:
Without this comment, JSHint will get mad that we're using the global variables `describe`, `it`, etc. that are required by Jasmine.